website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -1,110 +1,122 @@
 
-3. Navigate to [your profile](https://cloud.getdbt.com/settings/profile) and enable the **Beta** flag under **Experimental Features.**
+Note that if you are using VS Code, you'll need to restart it to pick up modified environment variables.
+:::
 
-4. Create an environment variable with your [dbt Cloud API key](https://cloud.getdbt.com/settings/profile#api-access):
+3. Verify the installation by running `./dbt --help` from the command line. If the help text does not indicate that you're using the dbt Cloud CLI, make sure you've deactivated your pyenv or venv and do not have a version of dbt globally installed.
 
-```bash
-vi ~/.zshrc
+#### Updating your dbt Cloud installation - Windows
 
-# dbt Cloud CLI
-export DBT_CLOUD_API_KEY="1234" # Replace "1234" with your API key   
-```
+Follow the same process in [Installing dbt Cloud CLI](#manually-install---windows) and replace the existing `dbt.exe` executable with the new one. During the beta period, we recommend updating before filing a bug report, as the API is subject to breaking changes!
 
-5. Load the new environment variable. Note: You may need to reactivate your Python virtual environment after sourcing your shell's dot file. Alternatively, restart your shell instead of sourcing the shell's dot file
+### Manually install - Linux 
 
-```bash
-source ~/.zshrc
-```
+1. Download the latest Linux release for your platform from [GitHub](https://github.com/dbt-labs/dbt-cli/releases). (Pick the file based on your CPU architecture)
 
-6. Navigate to a dbt project
+2. Extract the `dbt-cloud-cli` binary to the same folder as your dbt project.
 
 ```bash
-cd ~/dbt-projects/jaffle_shop
+tar -xf dbt_0.29.9_linux_amd64.tar.gz
+./dbt --version
 ```
 
-7. Create a `dbt_cloud.yml` in the root project directory. The file is required to have a `project-id` field with a valid [project ID](#glossary). Enter the following commands:
+:::info
 
-```bash
-pwd # Input
-/Users/user/dbt-projects/jaffle_shop # Output
-```
+Advanced users can configure multiple projects to use the same Cloud CLI executeable by adding it to their PATH environment variable in their shell profile.
 
-```bash
-echo "project-id: '<your project id>'" > dbt_cloud.yml # Input
-```
+:::
+
+3. Run the following command to verify the installation:
 
 ```bash
-cat dbt_cloud.yml # Input
-project-id: '123456' # Output 
+./dbt --help
 ```
 
-You can find your project ID by selecting your project and clicking on **Develop** in the navigation bar. Your project ID is the number in the URL: https://cloud.getdbt.com/develop/26228/projects/PROJECT_ID.
+- If the help text doesn't show you're using the dbt Cloud CLI, ensure you've deactivated pyenv or venv and don't have a global dbt version installed
 
-If `dbt_cloud.yml` already exists, edit the file, and verify the project ID field uses a valid project ID.
-
-#### Upgrade the CLI with Brew
-
-```bash
-brew update
-brew upgrade dbt-cloud-cli
-```
+## Update dbt Cloud CLI
```

Comment:
any manual update for linux?

Reply:
Same as windows - redownload via the link